docs/how-to-guides/blob-migration.md

```diff
@@ -0,0 +1,77 @@
+        "StartCopy": false,
+        "StartDelete": false,
+        "CopyFileOperationId": "1d4689da-ca3b-4659-b0c7-7bf6c9ff25e1",
+        "DeleteFileOperationId": "ce38a27e-b194-4645-b47a-fe91c38c330f",
+        "CleanupDeletedFileOperationId": "d32a0469-9c27-4df3-a1e8-12f7f8fecbc8",
+        "CleanupFilterTimeStamp": "2022-08-01"
+      }
+    }
+}
+```
+
+The same settings can be adjusted as part of the `App Service` Azure Service's Configuration settings if you used the [Deploy to Azure option from our README.](https://github.com/microsoft/dicom-server#deploy-to-azure) :
+
+![app-service-settings-configuration](../images/app-service-settings-configuration.png)
+
+
+## Migration steps
+
+### If deployed new service and have not created any files yet
+1. [You can upgrade to the latest version of the service](../resources/dicom-server-maintaince-guide.md) and skip the migration steps. The configuration section `Blob Migration` will not be present when service is on the latest version.
+
+### If you have already uploaded DICOM files but do not care about migrating the data
+1. If you have already uploaded DICOM files but do not care about migrating the data, you can use the [Delete API](../resources/conformance-statement.md#delete) to delete all existing studies.
+2. [You can upgrade to the latest version of the service](../resources/dicom-server-maintaince-guide.md) and skip the migration steps. The configuration section `Blob Migration` will not be present when service is on the latest version.
+
+### If you have already uploaded DICOM files and want to migrate the data
+If you have already uploaded DICOM files and want to migrate the data, you will need to execute the following steps before upgrading. This scenario has two options depending on whether if you want interruption to the service or not. Make sure Azure Monitor is configured to monitor the service before starting the migration (for more info on how to configure Azure monitor, please refer to [the Azure Monitor guide](../how-to-guides/configure-dicom-server-settings.md#azure-monitor)).
+
+#### With Service Interruption
+If you are ok with interruption to the service, you can follow the steps below. The interruption here is a self-managed one and using the service while the copying is occuring and before switching to the new format can corrupt the data path or retrieve wrong data.
+
+1. Set `BlobMigration.StartCopy` to `true` and restart the service.
+   1. When restarting the service, ensure it is in such a way that it picks up the new application settings and this will vary by how your service is deployed.
+   2. This will trigger the `CopyFiles` Durable Function which will copy the old format DICOM files to the new format.
+   3**Do not use the service at this time**. We want to make sure all files are copied over.
+2. To ensure Copy has been completed, you can check Azure Monitor logs for `"Completed copying files."` message. This will indicate that the copy has been completed.
```

Comment:
```suggestion
2. To ensure the Copy operation has been completed, you can check Azure Monitor logs for a `"Completed copying files."` message. This will indicate that the operation has been completed:
```